login-form/src/components/page-components/Dashboard.js

```diff
@@ -59,14 +59,34 @@ const Dashboard = ({
   );
 };
 
+const selectEmail = (state) => {
```

Comment:
have these selectors in a separate file. Ideally, there should be a folder called selectors and inside that, you should create selector files using proper names.